packages/main/src/plugin/container-registry.ts

```diff
@@ -1941,7 +1941,20 @@ export class ContainerProviderRegistry {
     let telemetryOptions = {};
     try {
       let container: Dockerode.Container;
-      if (options.pod) {
+      let forceLibPod = false;
+
+      // the device option requesting an nvidia gpu on linux only works
+      // if the LibPod API is used. Check if such a device is requested
+      // and if so force the use of LibPod
+      if (options.HostConfig?.Devices) {
+        for (const device of options.HostConfig?.Devices ?? []) {
```

Comment:
hello, I'm not sure why we're checking if there is Devices and then on the next line, we also assume it can be empty and we default to []

looks like the if condition is useless

Reply:
Good point, will see if I can remove it

Reply:
@benoitf removed the extra line and also rebased.